internal/cmd/functional-benchmarker/main.go

```diff
@@ -12,18 +12,23 @@ import (
 	"github.com/openshift/cluster-logging-operator/internal/constants"
 	"github.com/openshift/cluster-logging-operator/test/helpers/oc"
 
-	log "github.com/ViaQ/logerr/v2/log/static"
 	"github.com/openshift/cluster-logging-operator/internal/cmd/functional-benchmarker/config"
 	"github.com/openshift/cluster-logging-operator/internal/cmd/functional-benchmarker/reports"
 	"github.com/openshift/cluster-logging-operator/internal/cmd/functional-benchmarker/runners"
 	"github.com/openshift/cluster-logging-operator/internal/cmd/functional-benchmarker/stats"
 	"github.com/openshift/cluster-logging-operator/internal/utils"
 )
 
+var (
+	log = utils.InitLogger("functional-benchmarker")
```

Comment:
Why aren't you using the `InitStaticLogger()` function here when you use it in previous files?